cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -17,13 +15,8 @@
 ]
```

Comment:
Ah, yes! This was the other case where linter would complain. So again, if I remove it from params, do we have a way to trick linter?